logstash-core/src/main/java/org/logstash/secret/store/backend/JavaKeyStore.java

```diff
@@ -51,6 +51,7 @@
     private Lock writeLock;
     //package private for testing
     static String filePermissions = "rw-rw----";
+    private static boolean isWindows = System.getProperty("os.name").startsWith("Windows");
```

Comment:
@jakelandis can be `final`

Reply:
@jakelandis also since it's a constant `IS_WINDOWS` would be better :)

Reply:
done and done.